src/app/containers/StoryPromo/index.jsx

```diff
@@ -58,12 +60,30 @@ StoryPromoImage.propTypes = {
   imageValues: shape(storyItem.indexImage).isRequired,
 };
 
+const LiveComponent = ({ headline, service, dir }) => (
+  // eslint-disable-next-line jsx-a11y/aria-role
+  <span role="text">
+    <LiveLabel service={service} dir={dir}>
+      LIVE
+    </LiveLabel>
+    <VisuallyHiddenText lang="en-GB">Live, </VisuallyHiddenText>
```

Comment:
Is there a translation for this? Is there an issue to add it?

Reply:
This is a deliberate design choice not to translate the LIVE label.

Reply:
Awesome :)